java/list-related-features/README.md

```diff
@@ -1,20 +1,30 @@
-# List Related Features
-Shows related features to any feature tapped on.
+# List related features
 
-![List Related Features App](list-related-features.png)
+List features related to the selected feature.
+
+![Image of list related features](list-related-features.png)
+
+## Use case
+
+Related features are useful for managing relational information, like what you would store in a relational database management system (RDBMS). You can define relationship between records as one-to-one, one-to-many, or many-to-one. For example, you could model inspections and facilities as a many-to-one relationship. Then, for any facility feature, you could list related inspection features.
 
 ## How to use the sample
-The List Related Features app has a `MapView` loaded with a web map `FeatureLayer`. When you tap on a `Feature` you get a returned list of related features in an Android `BottomSheet`. The `FeatureLayer` has more than one relationship, the `ListView` results for each relationship.
+
+Tap on a feature to select it. The related features will be displayed in a list.
 
 ## How it works
-The `FeatureTable` from a selected `Feature` is used as a parameter on a `FeatureTable.queryRelatedFeaturesAsync` method.  The results returned represent related `Feature`s of which you can obtain the `ArcGISFeatureLayerInfo` to get the field used to filter the attribute values and notify the `ArrayAdapter` of the changes.
+
+1. With a `Feature`, call `queryRelatedFeaturesAsync` on the feature's feature table.
+2. Iterate over the result's collection of `RelatedFeatureQueryResult` objects to get the related features and add them to a list.
 
 ## Relevant API
+
 * ArcGISFeature
-* ArcGISFeatureLayerInfo
 * ArcGISFeatureTable
+* FeatureQueryResult
+* FeatureTable.QueryRelatedFeatures
```

Comment:
```suggestion
* ArcGISFeatureTable.queryRelatedFeatures
```

Reply:
they are used in the DatumTransformationAdapter.java file.